config/rule.go

```diff
@@ -44,6 +50,8 @@ func (at *ActionType) MarshalJSON() ([]byte, error) {
 	case ACTION_FORWARD:
 		return []byte("\"FORWARD\""), nil
 	default:
-		return nil, fmt.Errorf("invalid action type: %d", *at)
+		//return nil, fmt.Errorf("invalid action type: %d", *at)
+		return nil, log.Info("invalid action type", zap.String(":", *at))
```

Comment:
All returning `fmt.Errorf()` should be kept AS-IS. They are returning an `error` type, not printing to log.